blockchain/evm/evm.go

```diff
@@ -1,20 +1,18 @@
-package blockchain
+package evm
```

Comment:
Since a lot of blockchains are EVM based, having an EVM package allows other blockchains to import the common functionality that they need.